inspectit-gepard-agent/src/main/java/rocks/inspectit/gepard/agent/internal/properties/PropertiesResolver.java

```diff
@@ -90,4 +97,41 @@ public static Duration getPollingInterval() {
         ? Duration.parse(pollingIntervalEnvProperty)
         : Duration.ofSeconds(10);
   }
+
+  /**
+   * Get the attributes which should be sent to the configuration server. When an attribute is
+   * configured via both system properties and environmental properties, the system property will be
+   * used.
+   *
+   * @return the attributes to be sent to the configuration server
+   */
+  public static Map<String, String> getAttributes() {
+
+    Map<String, String> attributes = new HashMap<>(getAttributesFromEnv());
+
+    // Override with system properties where available
+    attributes.putAll(getAttributesFromSystemProperties());
```

Comment:
Should we move this into a try-catch? If something goes wrong, the whole agent cannot start, because the static `AgentInfo.INFO` cannot be created. If something bad happens, return an empty map

Reply:
Hmm okey, I can do that! But should we then also add something like that to the other getters?